.azure-pipelines/publish.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+pr: none
+
+pool:
+  vmImage: "ubuntu-latest"
+
+steps:
+  - template: ./steps/build-test.yml
+
+  - script: |
```

Comment:
I based this on how we publish `autorest.go`: https://github.com/Azure/autorest.go/blob/main/eng/pipelines/publish-release-go.yml